test/LaTeXTest.py

```diff
@@ -0,0 +1,29 @@
+from MediawikiTest import MediawikiBase
+
+class LaTeXTest(MediawikiBase):
+    LATEX_URL = 'http://latexml:8080'
+    
+    def test_1(self):
```

Comment:
Suggestion to use more speaking test function names. They might be the only thing visible on the testrunner.